staubli_resources/urdf/common_colours.xacro

```diff
@@ -2,9 +2,15 @@
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
 
   <!-- RAL 1028 -->
-  <xacro:property name="colour_staubli_yellow" value="1.0 0.608 0 1" />
+  <xacro:property name="melon_yellow" value="1.0 0.608 0.0 1.0" />
+
+  <!-- RAL 1018 -->
+  <xacro:property name="zinc_yellow" value="0.973 0.953 0.208 1.0" />
 
   <!-- RAL 9016 -->
-  <xacro:property name="colour_staubli_white" value="0.945 0.941 0.918 1" />
+  <xacro:property name="traffic_white" value="0.965 0.965 0.965 1.0" />
+	
+  <!-- RAL 7043 -->
+  <xacro:property name="traffic_grey_b" value="0.306 0.329 0.322 1.0" />
 
 </robot>
```

Comment:
I actually calculated those values based on the information I found at http://www.ralcolor.com/ (i.e., `RGB (approx.)` column values divided by 255).